netlify/pages/pricing.yaml

```
@@ -170,8 +154,8 @@ docPlans:
       title: Get Started
       large: true
     - color: orange
-      href: "/demo"
-      title: Contact Us
+      href: "/demo?utm_source=documentation&utm_medium=button&utm_campaign=bookdemo"
```

I think source should be pricing, right?
```suggestion
      href: "/demo?utm_source=pricing&utm_medium=button&utm_campaign=bookdemo"
```